<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Action_FollowP2</class>
 <widget class="QWidget" name="Action_FollowP2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lab_title">
   <property name="geometry">
    <rect>
     <x>1525</x>
     <y>80</y>
     <width>390</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(5,40,87);
color: rgb(255, 255, 255);
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string> 第1课 擒敌拳</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_text">
   <property name="geometry">
    <rect>
     <x>1528</x>
     <y>505</y>
     <width>390</width>
     <height>575</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(5,40,87);
color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="control_frame">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>10</y>
     <width>371</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
}
QPushButton:hover{
border:none;
background-color: rgba(255, 255, 255, 50);
}

QFrame{
background-color: rgba(5,40,87,0);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_stoporplay">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../res.qrc">
        <normaloff>:/icons/figs/icon/menu_stop.png</normaloff>:/icons/figs/icon/menu_stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_replay">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../res.qrc">
        <normaloff>:/background/figs/icon/重新加载.png</normaloff>:/background/figs/icon/重新加载.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_exit">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../res.qrc">
        <normaloff>:/icons/figs/icon/menu_exit.png</normaloff>:/icons/figs/icon/menu_exit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="pose_frame">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>120</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius:10px;
border-top-left-radius:10px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
